Jenkinsfile.rh.release

```diff
@@ -24,6 +24,46 @@ properties([
   ]),
 ])
 
+String version = ''
+String checksum = ''
+
+dockerizedBuildPipeline(
+  deployBranch: 'main',
+  prepare: {
+    githubStatusUpdate('pending')
+    version = getVersionFromBuildName(env.releaseBuild_NAME)
+    checksum = readBuildArtifact('insight/insight-brain/release', env.releaseBuild_NUMBER, "artifacts/nexus-iq-server-${version}-bundle.tar.gz.sha256").trim()
+    updateIQServerVersionAndChecksum(version, checksum)
+    commitAndPushChanges(version)
+  }
+)
+
+void updateIQServerVersionAndChecksum(String version, String checksum) {
```

Comment:
Looks correct - it will be the same as https://github.com/sonatype/docker-nexus-iq-server/blob/master/Jenkinsfile#L275